frontend/src/components/smart/SkillBalanceDisplay.vue

```diff
@@ -6,8 +6,12 @@
     </div>
 
     <div class="balance-container">
-      <strong class="mr-2 balance-text">Balance</strong>
-      <span class="balance">{{ formattedSkillBalance }}</span>
+      <strong class="mr-2 balance-text">Total Balance</strong>
+      <span class="balance"
+        v-tooltip="{ content: totalSkillTooltipHtml , trigger: (isMobile() ? 'click' : 'hover') }"
+        @mouseover="hover = !isMobile() || true"
+        @mouseleave="hover = !isMobile() || false"
```

Comment:
|| false is redundant

Reply:
Removed || false from this and from WeaponIcon.vue as well